src/CostManagement/Changelog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Used the latest version of autorest to generate code
```

Comment:
Please be more specific, for example, "Fixed an error when user ..." or "Supported (some new feature)". Thanks

Reply:
Got it,thanks for your reminder.